src/main/java/org/apache/sling/resourceresolver/impl/mapping/MapEntries.java

```diff
@@ -329,7 +329,8 @@ private void execute() {
             try (ResourceResolver resolver = factory
                     .getServiceResourceResolver(factory.getServiceUserAuthenticationInfo("mapping"))) {
 
-                log.debug("vanity path init - start");
+                long initStart = System.nanoTime();
+                log.info("vanity path initialization - start");
```

Comment:
I don't think that this message is really helpful on INFO level, because it will be always started along the SlingRepository.
Leaving on DEBUG would be fine for me.

Reply:
Ack.